android/app/src/main/kotlin/io/nedaa/nedaaApp/AllPrayersWidget.kt

```diff
@@ -125,24 +124,27 @@ class AllPrayersWidget : GlanceAppWidget() {
     }
 
     private fun isSystemInDarkTheme(context: Context): Boolean {
-        return context.resources.configuration.uiMode and Configuration.UI_MODE_NIGHT_MASK == Configuration.UI_MODE_NIGHT_YES
+        return context.resources.configuration.uiMode and Configuration.UI_MODE_NIGHT_MASK ==
+                Configuration.UI_MODE_NIGHT_YES
     }
 
-
     private fun getTranslation(context: Context, message: String, date: ZonedDateTime?): String {
-        val resourceId = when (message) {
-            "fajr" -> R.string.fajr
-            "sunrise" -> R.string.sunrise
-            "dhuhr" -> if (duhurOrJumah(date)) R.string.jumaa else R.string.dhuhr
-            "asr" -> R.string.asr
-            "maghrib" -> R.string.maghrib
-            "isha" -> R.string.isha
-            "enable_exact_alarm_permission_message" -> R.string.enable_exact_alarm_permission_message
-            "allow" -> R.string.allow
-            else -> {
-                R.string.empty
+        val resourceId =
```

Comment:
this seems like formatted only, not sure if I'm missing something, but noticed several formatted lines changes

did we change formatter?

Reply:
Yes it formatted the only change for the widget is its now clickable to open the app.